src/pages/your-team.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import styled from 'styled-components'
+import media from 'styled-media-query'
+import { graphql } from 'gatsby'
+import SEO from '../components/seo'
+import PageContainer from '../components/page-container'
+import TeamMember from '../components/team-member'
+
+const About = props => {
+  return (
+    <>
+      <SEO title="Your Team" />
+      <PageContainer>
+        <Container>
+          <div className="heading">
+            <h1>Core Team</h1>
+          </div>
+          <p>
+            Our team of experts are deeply invested in the technology & design
+            community. We are passionate about delivering business value using
+            our experience of working on projects for companies like: Visa, The
+            Telegraph, PureGym, Fidelity and many more.
+          </p>
+        </Container>
+        <TeamContainer>
+          <TeamMember
+            image={props.data.mitch}
+            firstName="Mitch"
+            jobRole="Founder & Developer"
+            links={{
+              medium: 'https://medium.com/@mitchclay0',
+              linkedin: 'https://www.linkedin.com/in/mitchclay/',
+              twitter: 'https://twitter.com/MitchClay0',
+            }}
+          />
+          <TeamMember
```

Comment:
@deadlybyte @Corporalis Please comment any other links you'd like to add

Reply:
Nope, got all my links covered.